html/changelogs/Sindorman-bugfixes.yml

```diff
@@ -0,0 +1,8 @@
+author: PoZe
+
+delete-after: True
+
+changes:
+  - bugfix: "Mechs that were IONed and got into maintenance mode while having it forbidden to switch the mode can now be unlocked by their DNA owners. By being able to allow maintenance mode."
```

Comment:
Ion is not an acronym. Lowercase the word `ioned`. Might want to replace with `that were shot with an ion gun`.